doc/classes/AtlasTexture.xml

```diff
@@ -54,10 +57,13 @@
 	</methods>
 	<members>
 		<member name="atlas" type="Texture" setter="set_atlas" getter="get_atlas" brief="">
+			The texture that contains the atlas. Can be any Texture subtype.
 		</member>
 		<member name="margin" type="Rect2" setter="set_margin" getter="get_margin" brief="">
+			The margin around the region. The Rect2's 'w' (width) and 'h' (height) resize the texture so it fits within the margin.
```

Comment:
Link to [Rect2] class.  Also, Rect2 doesn't have w/h member variables, it has `size`.